packages/wrangler/src/config/validation.ts

```diff
@@ -2037,6 +2045,40 @@ const validateCflogfwdrBinding: ValidatorFn = (diagnostics, field, value) => {
 	return isValid;
 };
 
+const validateAssetsConfig: ValidatorFn = (diagnostics, field, value) => {
+	if (value === undefined) {
+		return true;
+	}
+	if (!Array.isArray(value) || value === null) {
+		return false;
+	}
+	let isValid = true;
+	for (const assetConfig of value) {
+		if (typeof assetConfig !== "object") {
+			diagnostics.errors.push(
+				`Expected the \`experimental_assets\` field to be an object, but got ${typeof value}.`
```

Comment:
```suggestion
				`The field \`experimental_assets\` should be an object, but got ${typeof value}.`
```

Reply:
to keep things consistent with other validation errors

Reply:
though, reading through the code further, there doesn't seem to be much consistency across validation copy :(